content_scripts/link_hints.coffee

```diff
@@ -625,6 +625,9 @@ LocalHints =
           if element.clientHeight < element.scrollHeight and Scroller.isScrollableElement element
             reason = "Scroll."
 
+      when "pre"
+        isClickable ||= !!(element.clientWidth - element.scrollWidth) || !!(element.clientHeight - element.scrollHeight)
+
```

Comment:
Got it, no problem. I will update the pr by your suggest to generalize the case with tests covered, thanks!